resources/design-document.md

```diff
@@ -86,30 +86,30 @@ _I will store Medications in a dynamoDbTable. I will store Notifications in a dy
 String customerId;
 String medName;
 String medInfo;
-Set<String> notificationTime;
+Set<Notification> notifications;
```

Comment:
Hmm, this is an interesting one. The general idea of the public/API models is that they are the representation of our entities that we return to API clients. So, I think the "correct" thing would be to map the DynamoDB-annotated Notification objects to NotificationModel objects. Functionally, in your case, it won't make any different, since the 2 classes have all the same fields.

Reply:
Ok thanks, I will update the list to a Set<NotificaitonModel>